test/test.test.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// The module 'assert' provides assertion methods from node
+import * as assert from 'assert';
+
+suite("Tests", () => {
+    test("test", async () => {
+        assert.equal(true, true);
```

Comment:
What does this test accomplish exactly?

Reply:
Just tests that tests are actually running.  Should be removed when we have actual tests in place.

Reply:
Makes sense to me. Thanks for the clarification!

Reply:
Thanks for looking.